### ISSUE TITLE
 UserWarning: The epoch parameter in `scheduler.step()` was not necessary and is being deprecated where possible. Please use `scheduler.step()` to step the scheduler. During the deprecation, if epoch is different from None, the closed form is used instead of the new chainable form, where available. Please open an issue if you are unable to replicate your use case: https://github.com/pytorch/pytorch/issues/new/choose.   warnings.warn(EPOCH_DEPRECATION_WARNING, UserWarning)

### ISSUE THREAD

**Issue by ZengcanXUE** [pytorch/pytorch#54390]
### Please note that this issue tracker is not a help form and this issue will be closed.

We have a set of [listed resources available on the website](https://pytorch.org/resources). Our primary means of support is our discussion forum:

- [Discussion Forum](https://discuss.pytorch.org/)

**Comment by skyline75489** [comment]
The warning is expected. What are you trying to solve here?

Check out [提问的智慧](https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/main/README-zh_CN.md)/[How To Ask Questions The Smart Way](http://www.catb.org/~esr/faqs/smart-questions.html) to learn how to ask a good question.
